src/main/java/ticTacToe/game/Mark.java

```diff
@@ -23,6 +27,11 @@ public static Mark create(String markChosen) {
         }
     }
 
+    public static Mark random() {
+        List<Mark> marks = Arrays.asList(Mark.values());
+        return marks.get(new Random().nextInt(marks.size()));
+    }
+
     public Mark doSwitch() {
```

Comment:
I'd call this method just `switch()` as switch is a verb already.

Reply:
Looks like `switch()` is a keyword and it won't allow me to use it. I'll call it `opponent()` as suggested by Felipe 🙂

Reply:
Or do you have better ideas maybe? I wish I could call it `switch()`...

Reply:
oh right! That's why you put the `do` in front of it. Maybe `swap()`?

Reply:
😍